### PR TITLE
Add "return widget" to Layout's add_widget() method to make chaining easier

### PR DESCRIPTION
**Pull request by cltrudeau**
What does this implement/fix?
-----------------------------
One line change that adds "return widget" to the end of Layout's add_widget() method. This allows doing one-line layout and creation such as:

```
self.my_button = layout.add_widget(Button("My Button"), 1)
```

### CODE REVIEW

**Comment by cltrudeau** [comment]
Hey Peter,

Something weird going on with the build. Fails just for Linux Py3.7. Works find in 3.6 and locally in 3.9. Doesn't seem to be complaining about my change, but something else.

Send me a comment if this is somehow my fault and I'll fix if I can.
...ct

**Comment by peterbrittain** [comment]
Interesting.  Looks like a failure in pip... for a package we shouldn't need anymore!  Happy for you to remove the appveyor package as part of this MR.

File: asciimatics/widgets/layout.py [hunk 1]
```diff
@@ -108,6 +108,8 @@ def add_widget(self, widget, column=0):
         if widget.name in self._frame.data:
             widget.value = self._frame.data[widget.name]
 
+        return widget
+
     def clear_widgets(self):
```
Comment (peterbrittain) [inline asciimatics/widgets/layout.py:113]: Can you please add a comment about the return to the function too?

**Review by peterbrittain — CHANGES_REQUESTED** [review]
When you said chaining, I thought you meant some way of calling add_widget again and again in the same line of code.  Got me wondering about a more functional API here...  But I think that would be a major overhaul and so not worth exploring here.

I like the way that this makes a mainline case easier...  Nice job!

Could you also tweak the UTs to test this in one case?

**Comment by cltrudeau** [comment]
> Interesting. Looks like a failure in pip... for a package we shouldn't need anymore! Happy for you to remove the appveyor package as part of this MR.

Only reference to that that I found was in a requirements.txt file. I removed it, but the problem is still happening.

Other changes have been made as requested.

**Comment by cltrudeau** [comment]
Hey Peter,

Just curious when the next release will be with the changes to TextBox and other things. No hurry, I've copied the widget into another project, but when the release happens want to get rid of the local copy.

**Comment by peterbrittain** [comment]
Soon.  I don't have any more major updates I want to get in now.

**Comment by peterbrittain** [comment]
Sigh.  Travis support's fix didn't work...  Let's see what they come up with next.

**Pull request merged**